src/SourceBuild/content/repo-projects/nuget-client.proj

```diff
@@ -10,6 +10,8 @@
     <BuildScript>$([MSBuild]::NormalizePath('$(ProjectDirectory)', 'eng', 'dotnet-build', 'build$(ShellExtension)'))</BuildScript>
     <!-- Passed for compatibility with new P4 SDK. -->
     <BuildArgs>$(BuildArgs) /p:GenerateResourceUsePreserializedResources=true</BuildArgs>
+    <!-- Passed for signing -->
+    <BuildArgs Condition="'$(Sign)' == 'true'">$(BuildArgs) /p:Sign=true</BuildArgs>
```

Comment:
Are you sure nuget supports this? The comment in L7 suggests otherwise.

Reply:
I just realized this... this will get handled by https://github.com/dotnet/source-build/issues/4065